src/components/JobConfiguration.vue

```diff
@@ -236,7 +237,11 @@ export default defineComponent({
       if (job?.status === 'SERVICE_DRAFT') {
         this.store.dispatch('job/scheduleService', job)
       } else if (job?.status === 'SERVICE_PENDING') {
-        this.store.dispatch('job/updateJob', job)
+        this.store.dispatch('job/updateJob', job).then( async (resp) => {
+          if(resp) {
+            await this.store.dispatch('job/fetchPendingJobs', { eComStoreId: this.getCurrentEComStore.productStoreId, viewSize: process.env.VUE_APP_VIEW_SIZE, viewIndex: 0 });
```

Comment:
I think we should use common `getPendingJobs` method

Reply:
Sir, we are dispatching this action from `jobConfiguration` component, and the method suggested by you is defined inside of `pipeline` page.